src/olympia/constants/permissions.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+
+
+AclPermission = namedtuple('AclPermission', 'app, action')
+
+# Special null rule.
+NONE = AclPermission('None', 'None')
+
+# Admin super powers.  Very few users will have this permission (2-3)
+ADMIN = AclPermission('Admin', '%')
+
+# Can view admin tools.
+ADMINTOOLS = AclPermission('AdminTools', 'View')

```

Comment:
I was aiming for a consistent `THEAPPNAME_THEACTIONNAME` for `TheAppName:TheActionName`, though I'm aware that's not consistent with python constant naming.  The extra `_`s means its not always possible to know if `FOO_BAR_BOO` is `FooBar:Boo` or `Foo:BarBoo` from just the constant name.  Is `ADMIN_TOOLS__VIEW` too wordy?